### ISSUE TITLE
membership for deleted orga is shown

### ISSUE THREAD

**Issue by davewood** [certat/do-portal#24]
if a organization is deleted you still see memberships referencing the organisation. this in turn breaks the other stuff

example:
https://<staging-server-cp-portal>/api/1.0/organization_memberships
returns

```
{"city": "Membership City 318", "country": {"cc": "AT", "id": 10, "name": "Austria"}, "country_id": 10, "email": "Membership-Email-318@example.com", "id": 318, "membership_role_id": 1, "mobile": "+43 2 318", "organization_id": 38, "phone": "+43 1 318", "sms_alerting": 0, "street": "Membership Street 318", "user_id": 183, "zip": "Membership ZIP 318"}
```

```
do_portal=# select * from organizations where id = 38;
-[ RECORD 1 ]---------+-----------------------------
created               | 2017-09-27 14:51:52.383778
updated               | 2017-10-24 08:08:18.119058
id                    | 38
organization_group_id | 
is_sla                | t
abbreviation          | Organization Abbreviation 38
old_ID                | 
full_name             | Organization Full Name 38
display_name          | Organization Display Name 38
mail_template         | EnglishReport
mail_times            | 3600
ts_deleted            | 2017-10-24 08:08:18.046181
deleted               | 1
parent_org_id         | 3
```

**Comment by davewood** [comment]
evt sollt ma wenn orga deleted wird ebenfalls membership als deleted markieren, so aehnlich wie in diesem commit: 5863708

**Comment by MarkHofstetter** [comment]
fixed  in 9bc485a886eb4e7b4e9a539c3381650f5a62e805
